hack/update-codegen.sh

```diff
@@ -94,7 +95,7 @@ function generateApplyConfiguration(){
 export CLIENTSET_PKG=clientset
 export CLIENTSET_NAME=versioned
 
-for group in apiserver apps authorization build cloudnetwork image imageregistry machineconfiguration oauth project quota route samples security securityinternal template user; do
+for group in apiserver apps authorization build cloudnetwork image imageregistry network oauth project quota route samples security securityinternal template user; do
```

Comment:
Why has this moved network up here? I wouldn't have expected that unless we removed an API version from the network group?

Reply:
I believe there were errors in the MCO with vendoring without this change. However, this could also be a copy/paste error here. I will remove it and see if it causes and issues on my end.